app/templates/activity.html

```diff
@@ -50,6 +50,9 @@
     {{ event.message }}
   {% elif event.type == 'user_joined_event' %}
     <p>{{ gettext('%(user)s joined NOI.', user=user_link(event.user)) }}</p>
+  {% elif event.type == 'connection_event' %}
+    <p>{{ gettext('+%(new)d new connections made in NOI.', new=event.emails.count()) }}</p>
+    <p>{{ gettext('+%(total)d connections!', total=event.total_connections) }}</p>
```

Comment:
![2015-10-02_15-43-27](https://cloud.githubusercontent.com/assets/124687/10256058/79b41532-691c-11e5-8506-585b589ca53f.png)

Based on the styling in the mock-ups, it seems like we'll want to wrap the new/total numbers in `<strong>` tags or something... One future-proofy way to do that without disrupting the l10n might be to actually change the `%d`'s to `%s`'s, so that they can take rich+safe HTML as arguments in the future?